src/features/orcidlink/HomeUnlinked.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Displays the "home page", or primary orcidlink view, for unlinked users.
+ */
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardHeader,
+  Typography,
+  Unstable_Grid2 as Grid,
+} from '@mui/material';
+
+export default function Unlinked() {
+  return (
+    <Grid container rowSpacing={2} columnSpacing={2}>
+      <Grid xs={6}>
+        <Card variant="outlined">
+          <CardHeader title="Create Your KBase ORCID® Link!" />
+          {/* 
+            A note on the padding and margin overrides. Mui is not very smart about components
+            embedded in card content. The card content has a padding on all subelements (header, 
+            content, actions) which is added to any margins and padding of components contained 
+            within, so it pretty easily blows up into a lot of blank space. 
+            To keep the empty space under control (and others have commented on this 
+            in GH issues), one must fiddle with padding and margins on a case-by-case
+            basis.
```

Comment:
Seems fine for now! @codytodonnell, do you have recommendations here? You are more familiar than I...

Reply:
It sounds like you're bumping against some known issues so overrides make sense. If you have to repeat the overrides you could either create a wrapper component or make a global `styleOverride` for the `Card` component in the MUI theme object. 

Another possibility could be to wrap the inner `CardContent` in a `Stack`. I'm not sure but that could offset the subelement padding being added by Card. It may also help to omit using the `CardHeader` and just include the title in the `CardContent`.

Reply:
This may be partly addressed in MUI v6: https://github.com/mui/material-ui/pull/32471
I've made some adjustments, after which it seems to make more sense to make overrides solely to the content to remove padding and/or margin from the top and bottom of the content

Reply:
I ended up just removing the removal of padding.